clients/hadoopfs/src/test/java/io/lakefs/LakeFSFileSystemTest.java

```diff
@@ -141,6 +151,27 @@ public void setUp() throws Exception {
         fs.initializeWithClient(new URI("lakefs://repo/main/file.txt"), conf, lfsClient);
     }
 
+    /**
+     * @return all pathnames under s3Prefix that start with prefix.  (Obvious not scalable!)
+     */
+    protected List<String> getS3FilesByPrefix(String prefix) throws IOException {
+        final int maxKeys = 1500;
+
+        ListObjectsRequest req = new ListObjectsRequest().
+            withBucketName(s3Bucket).
+            withPrefix(prefix)
+            .withMaxKeys(maxKeys);
+        ObjectListing listing = s3Client.listObjects(req);
+        if (listing.isTruncated()) {
+            Assert.fail(String.format("[internal] no support for test that creates >%d S3 objects", maxKeys));
+        }
+
+        return listing.getObjectSummaries().
+            stream().
+            map(summary -> summary.getKey()).
+            collect(Collectors.toList());
```

Comment:
Consider
```java
return Lists.transform(listing.getObjectSummaries(), S3ObjectSummary::getKey);
```

Reply:
Yikes, thanks!  Forgot that (lazy) "map" is called "transform" in *Google*'s Java library!

Also thanks for reminding me of the one place Java forgot its syntax and decided to use C++ `::`.